### ISSUE TITLE
Incorrect message from status check with regard to domain's MX record

### ISSUE THREAD

**Issue by mathuin** [mail-in-a-box/mailinabox#329]
Here is the message in question:

````
This domain's DNS MX record is incorrect. It is currently set to '2 box.mathuin.org' but should be '10 box.mathuin.org'. Mail will not be delivered to this box. It may take several hours for public DNS to update after a change. This problem may result from other issues listed here.
````

The message is incorrect because mail will be delivered to this box.

The MX record check should compare the hostname with the lowest priority to the box's hostname instead of performing a literal string comparison to '10 hostname'.

**Comment by mathuin** [comment]
Just sent a pull request #332 to address this issue.

**Comment by JoshData** [comment]
Fixed in #332.
